packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer-card-header.jsx

```diff
@@ -0,0 +1,62 @@
+import classNames         from 'classnames';
+import PropTypes          from 'prop-types';
+import React              from 'react';
+import { Icon }           from '@deriv/components';
+import ContractCardHeader from 'App/Components/Elements/ContractCard/contract-card-header.jsx';
+import ContractTypeCell   from 'App/Components/Elements/PositionsDrawer/contract-type-cell.jsx';
+import ProgressSlider     from 'App/Components/Elements/PositionsDrawer/ProgressSlider';
+import Shortcode          from 'Modules/Reports/Helpers/shortcode';
+import { getCurrentTick } from 'Stores/Modules/Portfolio/Helpers/details';
+
+const CardHeader = ({
+    contract_info,
+    has_progress_slider,
+}) => {
+    const getTick = () => {
+        if (!contract_info.tick_count) return null;
+        let current_tick = getCurrentTick(contract_info);
+        current_tick = (current_tick > getCurrentTick(contract_info)) ?
```

Comment:
🤔 How can this be possible?

Reply:
@oskar-binary haha yea, we're confused as well 😅